iohub/cli/cli.py

```diff
@@ -87,3 +89,60 @@ def convert(input, output, grid_layout, chunks):
         chunks=chunks,
     )
     converter()
+
+
+@click.command()
+@click.help_option("-h", "--help")
+@click.argument(
+    "csvfile",
+    type=click.File("r"),
+)
+@click.argument(
+    "zarrfile",
+    type=click.Path(
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        resolve_path=True,
+        path_type=pathlib.Path,
+    ),
+)
+def rename_wells_cli(csvfile, zarrfile):
```

Comment:
```suggestion
def rename_wells(csvfile, zarrfile):
```
This is the CLI name, so I think you probably want `rename_wells`